Can't compile app on Windows machine

Issue(?): running `npm start` on a Windows 10 machine with Git Shell will result in an error, preventing the app from compiling.

Analysis:
* Error message received is:
```
ERROR in ./app/partials/footer-logo.cjsx
Module not found: Error: Cannot resolve 'file' or 'directory' ./../../node_modules/webpack/buildin/module.js in C:\projects\wildcam-gorongosa/app\partials
 @ ./app/partials/footer-logo.cjsx 1:0-57
```

So all this time, I've been running `wildcam-gorongosa` on my office Macbook, and it's been compiling/running fine. At this moment though, I'm running WG from my Windows 10 Surface Pro 3 Machine with Git Shell, and when I try to compile/run the app I receive the above error.

A quick trace revealed that in `app/partials/footer-logo.cjsx` there's this line of code...

`module?.exports = React.createClass`

that when changed to...

`module.exports = React.createClass`

will fix my Windows-specific issue.

I was able to properly test #222 by temporarily removing the `?`, but I'm curious to know why `module?.exports` exists, and only specific to `footer-logo.cjsx`. I have no idea what the `modules?.` does in this context as my Coffee-Fu is weak. (It's decaf, really.) @srallen , do you have any insights into this?

--- a/app/partials/footer-logo.cjsx
+++ b/app/partials/footer-logo.cjsx
@@ -1,6 +1,6 @@
 React = require 'react'
 
-module?.exports = React.createClass
+module.exports = React.createClass
   displayName: 'FooterLogo'
 
   render: ->

